variables.tf

```diff
@@ -4,7 +4,7 @@ variable "name" {
 }
 
 variable "vpc_id" {
-  description = "The VPC to deploy Enrich within"
+  description = "The VPC to deploy Enrich within (must have DNS hostnames enabled)"
```

Comment:
Why does it need the dns hostnames enabled?

Reply:
Because of [this line](https://github.com/snowplow/enrich/blob/master/modules/kinesis/src/main/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Source.scala#L111), we need to resolve `localhost`, and i found out that without DNS hostname it doesn't work